# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/generic/moz.build
# Commit: 5ded58486aba
# Full Hash: 5ded58486abab5712f305a88bbdd746ae74dcdef
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-03-06 09:45:20
# Description:
#   Bug 1819957 - Split out nsTextPaintStyle into its own source file. r=emilio
#   
#   This lets us instantiate it on the stack in nsDisplayText::Paint.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171635
# ==============================================================================

diff -r 58235f400c11 -r 5ded58486aba layout/generic/moz.build
--- a/layout/generic/moz.build	Sun Mar 05 20:37:07 2023 +0000
+++ b/layout/generic/moz.build	Sun Mar 05 22:24:10 2023 +0000
@@ -206,6 +206,7 @@
     "nsSubDocumentFrame.cpp",
     "nsTextFrame.cpp",
     "nsTextFrameUtils.cpp",
+    "nsTextPaintStyle.cpp",
     "nsTextRunTransformations.cpp",
     "nsVideoFrame.cpp",
     "PrintedSheetFrame.cpp",